test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+require 'pry'
+
+describe RecipesController do
+
+  describe 'index' do
+    it "should get index path" do
+      VCR.use_cassette("recipes") do
+        get recipes_path
```

Comment:
You should have this be:  `get recipes_path(search: "pasta")`

Then add a test for a search term which returns no results.